src/data/code-samples/en/accessibility-label/swift-ui.md

```diff
@@ -0,0 +1,31 @@
+# Accessibility label - SwiftUI
+
+In SwiftUI, the [`accessibilityLabel(_:)`](https://developer.apple.com/documentation/swiftui/view/accessibilitylabel(_:)-5f0zj) view modifier is used to assign an accessibility label to a view. It's essential to keep the accessibility label concise yet meaningful for optimal usability.
+
+```swift
+Link(destination: URL(string: "https://appt.org/en/")!, label: {
+    Text("appt.org")
+})
+.accessibilityLabel("Appt.org")
+```
+
+You can enhance accessibility in SwiftUI by using the [`Text`](https://developer.apple.com/documentation/swiftui/text/init(_:)-2tl9f) initializer with an `AttributedString` parameter, offering precise control over pronunciation. For instance, you can spell out each character using [`.accessibilitySpeechPunctuation`](https://developer.apple.com/documentation/foundation/nsattributedstring/key/1620201-accessibilityspeechpunctuation) or specify a language with [`.accessibilitySpeechLanguage`](https://developer.apple.com/documentation/foundation/nsattributedstring/key/1620188-accessibilityspeechlanguage).
```

Comment:
Can you add code sample right below this paragraph for attributedstring. (lines 17-21)
Also set the attributedString to a component, maybe `Text`?